docs/general/attribute-requirement-level.md

```diff
@@ -0,0 +1,76 @@
+# Attribute Requirement Levels for Semantic Conventions
+
+**Status**: [Stable](../document-status.md)
+
+<details>
+<summary>Table of Contents</summary>
+
+<!-- toc -->
+
+- [Required](#required)
+- [Conditionally Required](#conditionally-required)
+- [Recommended](#recommended)
+- [Opt-In](#opt-in)
+- [Performance suggestions](#performance-suggestions)
+
+<!-- tocstop -->
+
+</details>
+
+_This section applies to Log, Metric, Resource, and Span, and describes requirement levels for attributes defined in semantic conventions._
+
+Attribute requirement levels apply to the [instrumentation library](../glossary.md#instrumentation-library).
```

Comment:
Now we will need to use the full link to the spec as we do elsewhere in the semconv repo. (referring to the spec version)